src/ignore.rs

```diff
@@ -192,6 +195,11 @@ impl Core {
         Ok(())
     }
 
+    pub fn autodetect_templates(&self) -> Result<Vec<String>> {
+        let entries = read_dir(current_dir()?)?.map(|e| e.unwrap()).collect();
```

Comment:
I know `unwrap` is not safe but I was not able to come with a simple solution

Reply:
```suggestion
        let entries = read_dir(current_dir()?)?.filter_map(|e| e.ok()).collect();
```

would work I think.